Hunted Mobile/Hunted Mobile/Hunted Mobile/Model/Response/Json/EventDataModels.cs

```diff
@@ -12,6 +12,7 @@ public class EventData : JsonResponseData {
     public class IntervalEventData : EventData {
         public UserData[] users;
         public LootData[] loot;
+        public bool drone_is_active;
```

Comment:
Dit lijkt me niet helemaal volgens onze code guideliness

Reply:
De variabelen in die data classes/structs moeten precies hetzelfde zijn als de naam van de velden in de JSON data.

Reply:
Oooooh mijn fout, had niet gezien dat dit de api variables waren